src/dnmd/tables.c

```diff
@@ -630,3 +640,29 @@ bool consume_table_rows(mdtable_t* table, uint8_t const** data, size_t* data_len
     table->data.size = rows_len;
     return true;
 }
+
+bool table_is_indirect_table(mdtable_id_t table_id)
```

Comment:
Can we instead convert this to a fast lookup table?

`bool indirect_state[] = { false, ..., true, ..., false, ... };`

or perhaps a switch statement?

```c
switch (table_id)
{
    case ...:
        return true;
    default:
        return false;
}
```